src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  itemsOnPage: string[];
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  itemsOnPage,
+}) => {
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <Link
+          data-cy="prevLink"
+          className={cn('page-link')}
+          aria-disabled={currentPage === 1}
+          to={`?page=${currentPage - 1}&perPage=${perPage}`}
+          onClick={() => currentPage !== 1 && onPageChange(currentPage - 1)}
+        >
+          «
+        </Link>
+      </li>
+
+      {getNumbers(
+        1,
+        total % perPage === 0 ? total / perPage : total / perPage + 1,
```

Comment:
The calculation of the total number of pages is not encapsulated, which makes the code less readable and harder to maintain. Consider moving this logic into a separate function or variable with a descriptive name.